intellij-plugin-structure/tests/src/test/kotlin/com/jetbrains/plugin/structure/edu/mock/EduPluginMockTest.kt

```diff
@@ -222,6 +223,67 @@ class EduPluginMockTest(fileSystemType: FileSystemType) : BasePluginManagerTest<
     assertTrue(pluginCreationSuccess.warnings.isEmpty())
   }
 
+  @Test
+  fun `check presentable name for sections and lessons with custom name`() {
+    val lesson = EduItem(
+      type = ItemType.LESSON.id,
+      title = "lesson",
+      customName = "lesson custom name",
+    )
+    val section = EduItem(
+      type = ItemType.SECTION.id,
+      title = "section",
+      customName = "section custom name",
+      items = listOf(lesson)
+    )
+
+    val result = createPluginSuccessfully(buildEduPlugin(temporaryFolder.newFolder().resolve("course.zip")) {
+      items = listOf(section)
+    })
+    assertNotNull("Edu stats were not parsed", result.plugin.eduStat)
+    val stats = result.plugin.eduStat!!
+
+    assertEquals(
+      "Section's title should be equal to custom name if it's present",
+      section.customName, stats.sections.firstOrNull()?.title
+    )
+
+    assertEquals("Lesson's title should be equal to custom name if it's present",
+      lesson.customName, stats.lessons.firstOrNull()
+    )
+  }
+
+  @Test
```

Comment:
Maybe add a test with a default value for custom name? To check if there is no value pass for `EduItem`, `title` will be used

Reply:
I did it [here](https://github.com/JetBrains/intellij-plugin-verifier/pull/1039/files#diff-ec8e6629bd4740150875a4cdb1dab75e4adef20c7533eb8a1ced7951a748edf5R257), isn't it?